docs/9.0/api/contextMenu.md

```diff
@@ -13,7 +13,7 @@ editLink: false
 ## Description
 
 This plugin creates the Handsontable Context Menu. It allows to create a new row or column at any place in the
-grid among [other features](@/guides/accessories-and-menus/context-menu.md#context-menu-with-specific-options).
+grid among [other features](@/guides/accessories-and-menus/context-menu.md#contextmenuwithspecificoptions).
```

Comment:
This should point to the `/#context-menu-with-specific-options`.

Reply:
#8440